futures-util/src/future/either.rs

```diff
@@ -1,58 +0,0 @@
-use futures_core::{Future, Poll, Stream};
```

Comment:
This should've been deleted in an older PR, but it got left around accidentally.